pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -107,80 +105,6 @@ func getInfoInReg(regMatch *ofctrl.MatchField, rng *openflow13.NXRange) (uint32,
 	return regValue.Data, nil
 }
 
-// getNetworkPolicyInfo fills in tableName, npName, ofPriority, disposition of logInfo ob.
-func getNetworkPolicyInfo(pktIn *ofctrl.PacketIn, c *Controller, ob *logInfo) error {
```

Comment:
qq: why is this UT no longer needed? Is it because it is covered in some other testcases?

Reply:
Because the only usage of `getPacketInfo` was audit logging, a new function `ParsePacketIn` was later implemented in `pkg/ovs/openflow` that covered the exact functionality and was more widely used. 
Deleted the `getPacketInfo` function and thus the UT.

Reply:
I guess my question is then if the same test deleted here is already implemented for the UT of `ParsePacketIn`

Reply:
Looks like it was not implemented. Adding some similar UT for `ParsePacketIn` and other funcs in `ofctrl_packetin.go`.